app/models/support_ticket.rb

```diff
@@ -20,4 +20,16 @@ def close!(closer)
 
     update!(status: :closed, closer: closer, closed_at: Time.current)
   end
+
+  def category_s
```

Comment:
What does the `_s` represent?

Reply:
Ok, so I am not good with naming things :sweat_smile: 

Initially I thought about keeping these on a helper file, like
```
  category(ticket: ticket)
  
  def category(ticket:)
    ticket_types = {"survey_response" => "Support",
                    "admin" => "Admin",
                    "contact_info" => "Contact"}
    ticket_types[category]
  end
```

Then I thought have it in model it self(fat model)

At first I had it like `category_html`, like `.to_html` then moved to `category_s` like `.to_s` 

As I said, not good at naming :sweat_smile:

Reply:
I think moving it to the model was the right call for now. Mostly because I can't stand `helpers`, as they make code tightly coupled.

Ultimately, it would nice to set up a pattern for using view objects in this codebase. But I think it's been too small a project so far to considerate it too seriously. Since there is going to be a more presentational survey ticket form, #157 , it might be good to make some sort of view object for SupportTickets in that change.

For reference on my thoughts/opinions here, see [this article](https://jtway.co/cleaning-up-your-rails-views-with-view-objects-42cf048ea491) . [ViewComponent](https://github.com/github/view_component) might be a nice library we could use.

Reply:
Honestly, another lighter weight option could be adding `helper_method`s directly into the controller (instead of adding a global one into the helpers directory).

Reply:
They would then be scoped to the Tickets controller only.

Reply:
Yup, that is a solution, I will take it up as part of https://github.com/rubyforgood/inkind-admin/issues/157 
I hope that is ok.

Reply:
On a second thought, I think it might be ok to keep the existing setup, since we are using these method tied to the model for exporting the csv 

https://github.com/rubyforgood/inkind-admin/blob/main/app/controllers/admin/tickets_controller.rb#L11